micrometer-tracing/src/main/java/io/micrometer/tracing/Baggage.java

```diff
@@ -86,4 +100,23 @@ public BaggageInScope makeCurrent() {
      */
     BaggageInScope makeCurrent();
 
+    /**
+     * Sets the current baggage in scope with given value.
+     * @param value to set
+     * @return a {@link BaggageInScope} instance
+     */
+    default BaggageInScope makeCurrent(String value) {
```

Comment:
I'm debating with myself if the extra behavior (setting the value) should be called out in the method name or not. I like the overloads though.